src/css/hero.css

```diff
@@ -24,6 +24,20 @@
     overflow: hidden;
 }
 
+.mobile-hero-image::before {
```

Comment:
we should not have img in this, it is too much complicated, z-index should not exist, we have just background image to the hero section:
![image](https://github.com/user-attachments/assets/61328f80-3cf0-4c98-852f-ed65a5cf4f2a)

Reply:
for each type of device the img will be different (mobile, tablet, desktop), for each device the media query about resolution should be done and url do 2x screen (as in the screen above)

Reply:
you should remove all the images from hero section:
![image](https://github.com/user-attachments/assets/de290aa0-d47d-433b-9627-d829f5205e78)

pay also attention that is not properly formatted through Prettier